.env

```diff
@@ -1,8 +1,32 @@
 VUE_APP_DISPLAY_SCREENING_SCORES=true
-VUE_APP_WRITE_BACK_MODE=none
+
```

Comment:
We should NOT be committing a `.env` file to the repository.  Please add `.env` to the `.gitignore` and make a `default.env` file as a template, defining the known values and including all that need to be defined.

Reply:
@ivan-c  @achen2401  per our call just now